[rocprofiler-compute] Update Documentation for Strix Halo (#4794)
ROCm/rocm-systems: projects/rocprofiler-compute/docs/conceptual/cdna/system-speed-of-light.rst

.. meta::

>    :description: ROCm Compute Profiler performance model: System Speed-of-Light (CDNA)

Reviewer: `(CDNA)` is not added for other sections, we should be consistent.

>    :description: ROCm Compute Profiler performance model: System Speed-of-Light (CDNA)

Reviewer 2: ```suggestion
   :description: Learn about the System Speed‑of‑Light metrics for AMD Instinct MI GPUs using CDNA–CDNA4 architectures with ROCm Compute Profiler.
```

*********************

> System Speed-of-Light summarizes some of the key metrics from various sections

Reviewer 2: ```suggestion
The page lists the System Speed-of-Light metrics from various sections
```

> of ROCm Compute Profiler’s profiling report for **AMD Instinct™ MI-series**

Reviewer 2: ```suggestion
of the ROCm Compute Profiler’s profiling report for AMD Instinct™ MI-series
```

> accelerators (CDNA through CDNA4).

Reviewer 2: ```suggestion
GPUs based on CDNA-CDNA4 architectures.
```

> For **RDNA3.5 (gfx1151)** client GPUs, see :doc:`../rdna/system-speed-of-light`.

Reviewer 2: ```suggestion
For RDNA3.5 (gfx1151) client GPUs, see :doc:`../rdna/system-speed-of-light` under RDNA3.
```
